### ISSUE TITLE
Edit powers page rendering duplicates

### ISSUE THREAD

**Issue by zigdon** [zigdon/pacg-player#1]
Seems some of the powers are rendered multiple times, happens on multiple characters. The non-edit screen doesn't seem to have the issue.

![image](https://user-images.githubusercontent.com/44372/89352035-170ea500-d668-11ea-812b-8e89262ae19b.png)

**Comment by zigdon** [comment]
Seems to be coming from https://github.com/zigdon/pacg-player/blob/main/src/components/Common/CharacterSheet/SharedWithFunctions/upConvertPowers.ts#L63, but oof, that's hard to follow.

**Comment by waterfoul** [comment]
Yeah, the difference between the two has to deal with the fact that it doesn't try to match them up with the base until you open that modal. It took me a while to figure out how to match them up and can walk through what it does with you if you'd like
